runners/basic_runner.py

```diff
@@ -53,11 +53,14 @@ def __str__(self):
 
     def runnable_status(self):
         """
-        Always execute the task if `waf --options=run_always` was used
+        Always execute the test task
+
+        If the run_modified option is present, then only the modified test
+        binaries will be run.
         """
         ret = super(BasicRunner, self).runnable_status()
         if ret == Task.SKIP_ME:
-            if self.generator.bld.has_tool_option('run_always'):
+            if not self.generator.bld.has_tool_option('run_modified'):
```

Comment:
I want to make "run_always" the default behavior. So I can just write:
`python waf --run_tests`
The run_modified option is the opposite of run_always. You can use it to run only the modified test binaries (as described in the docstring above).
Btw, we can also get rid of run_modified and *always* execute the tests.